pkgs/development/tools/godot/mono/export-templates.nix

```diff
@@ -0,0 +1,14 @@
+{ godot-mono }:
+
+godot-mono.overrideAttrs (self: base: {
+  pname = "godot-mono-export-templates";
+  buildDescription = "mono export templates";
+  shouldBuildTools = false;
+  buildTarget = "release";
+  godotBinInstallPath = "share/godot/templates/${self.version}.stable";
```

Comment:
For the mono export templates, this should probably be `"share/godot/templates/${self.version}.stable.mono"`. That would be consistent with the directory name Godot uses for mono templates downloaded by the editor into `~/.local/share/godot/templates/`.

Reply:
Makes sense, I'll make this fix.